Kbs.Wpf/User/Read/Details/ReadDetailsUserPage.xaml.cs

```diff
@@ -57,5 +73,15 @@ private void Ban(object sender, RoutedEventArgs e)
                 _navigationManager.Navigate(() => new ReadDetailsUserPage(_navigationManager, userId));
             }
         }
+
+        private void RemoveMedal(int medalId)
+        {
+            MessageBoxResult result = MessageBox.Show("Weet u het zeker?", "Confirm", MessageBoxButton.YesNo, MessageBoxImage.Question);
```

Comment:
Verander de tekst naar "Weet u zeker dat u de medaille wilt verwijderen?":
```suggestion
            MessageBoxResult result = MessageBox.Show("Weet u het zeker dat u de medaille wilt verwijderen?", "Confirm", MessageBoxButton.YesNo, MessageBoxImage.Question);
```